vminfo_parser/analyzer.py

```diff
@@ -174,21 +175,6 @@ def calculate_disk_space_ranges(
 
         return disk_space_ranges_with_vms
 
-    def categorize_environment(self: t.Self, x: str, *args: str) -> str:
-        if pd.isnull(x):
-            return "non-prod"
-
-        if not args:
-            return "all envs"
-
-        # Ensure x is a string
-        if isinstance(x, str):
-            for arg in args:
-                if arg in x:
-                    return "prod"
-
-        return "non-prod"
-
```

Comment:
moved to vmdata

Reply:
What was the logic for moving this to vmdata vs analyzer? Perhaps I am still unclear as to the delineation between vmdata vs analyzer